src/Factories/ChoiceFactory.php

```diff
@@ -52,6 +52,21 @@ public function getMembershipCallToActionTemplate(): string {
 		throw new UnknownChoiceDefinition( 'Membership Call to Action Template configuration failure.' );
 	}
 
+	public function getAmountType(): string {
+		if ( $this->featureToggle->featureIsActive( 'campaigns.amount_options.5to300' ) ) {
+			return '5to300';
+		} elseif ( $this->featureToggle->featureIsActive( 'campaigns.amount_options.5to100' ) ) {
+			return '5to100';
+		} elseif ( $this->featureToggle->featureIsActive( 'campaigns.amount_options.15to250' ) ) {
+			return '15to250';
+		} elseif ( $this->featureToggle->featureIsActive( 'campaigns.amount_options.30to250' ) ) {
+			return '30to250';
+		} elseif ( $this->featureToggle->featureIsActive( 'campaigns.amount_options.50to500' ) ) {
+			return '50to500';
+		}
+		throw new UnknownChoiceDefinition( 'Amount option selection configuration failure.' );
```

Comment:
To prevent any more :baby::baby_bottle::baby: major boo-boos :baby::baby_bottle::baby:, I think we should return some default fallback value from the list above here. Probably also change it for all other functions in this file to be on the safe side.

Reply:
Since this A/B test is for the email campaign we don't have to configure `start date` in two different places => throwing an exception here is ok.